### PR TITLE
Ant design theme

### PR DESCRIPTION
**Pull request by hub-bla**

### CODE REVIEW

File: client/vite.config.ts [hunk 1]
```diff
@@ -1,7 +1,35 @@
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
-
+import { getThemeVariables } from 'antd/dist/theme';
```
Comment (Safeheaded) [inline client/vite.config.ts:3]: This import is not needed, please delete it.

**Review by Safeheaded — CHANGES_REQUESTED** [review]
Good job, everything works, please remove one line & we're good to go!

File: client/vite.config.ts (hunk 1 above)
Comment (hub-bla) [reply client/vite.config.ts]: Done!

**Review by Safeheaded — APPROVED** [review]
Perfecto

**Pull request merged**